lib/vquic/ngtcp2.c

```diff
@@ -858,8 +858,22 @@ static int ng_getsock(struct Curl_easy *data, struct connectdata *conn,
 
 static void qs_disconnect(struct quicsocket *qs)
 {
+  char buffer[256];
```

Comment:
It would be nice to extend this to at least NGTCP2_MAX_UDP_PAYLOAD_SIZE (1200).
RFC 9000 demands that Client Initial packet must be padded to 1200 bytes.  It looks like the current ngtcp2 code does not enforce this.

Reply:
fixed!